test/test-clusters/src/main/resources/fips/fips_java.policy

```diff
@@ -18,3 +18,8 @@ grant {
      permission org.bouncycastle.crypto.CryptoServicesPermission "exportPrivateKey";
      permission java.io.FilePermission "${javax.net.ssl.trustStore}", "read";
 };
+
+// rely on the caller's socket permissions, the JSSE TLS implementation here is always allowed to connect
+grant codeBase "file:${jdk.module.path}/bctls-fips-1.0.17.jar" {
```

Comment:
interesting, so this codebase path is actually working? 
I would be fine if we have it in grant all - it is a test cluster after all, but if you managed to make it work with that codebase path it is even better

Reply:
Since this file is passed to the jvm directly, it does not get the special system properties we set when loading plugin policy files. So the explicit jar name is necessary.